front/app/components/Author/index.tsx

```diff
@@ -97,7 +96,7 @@ const TimeAgo = styled.div`
   }
 `;
 
-export interface InputProps {
+export interface Props {
   authorId: string | null;
   createdAt?: string | undefined;
```

Comment:
It's already optional, no need to add `| undefined`. 
I know it's not something you added now, just noticed it.